apps/addons/tests/test_views.py

```diff
@@ -835,6 +835,11 @@ def test_more_url(self):
         eq_(pq(response.content)('#more-webpage').attr('data-more-url'),
             self.addon.get_url_path(more=True))
 
+    def test_unlisted_addon_returns_404(self):
+        """Unlisted addons are not listed and return 404."""
```

Comment:
I'd write this as `"""Unlisted addons are not listed (doh!) and return 404."""` :)